dbt/scripts/export_models.py

```diff
@@ -54,6 +42,12 @@ def parse_args():
         # get preserved
         formatter_class=argparse.RawTextHelpFormatter,
     )
+    parser.add_argument(
+        *constants.TARGET_ARGUMENT_ARGS, **constants.TARGET_ARGUMENT_KWARGS
+    )
+    parser.add_argument(
+        *constants.REBUILD_ARGUMENT_ARGS, **constants.REBUILD_ARGUMENT_KWARGS
+    )
```

Comment:
Although this wasn't in the scope of [the comment that inspired this PR](https://github.com/ccao-data/data-architecture/pull/586#discussion_r1750638576), I took the opportunity to factor out shared argparse argument definitions into the new `utils` module as well.